src/components/Molecules/ContentsLive.tsx

```diff
@@ -42,13 +43,71 @@ const LiveContents = () => {
         )
           .then((res) => res.json())
           .then((res) => res.contents)
-        const sortData = fetchDate.sort(compare).slice(0, 5)
-        setArticlesData(sortData)
-        setActiveLive(sortData[0])
+        const currentDate = new Date()
```

Comment:
テストに関しては取得した今日の日付を操作することで色々網羅できるかと。
```suggestion
        const currentDate = new Date()
        currentDate.setDate(currentDate.getDate() + 100)
```